addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -385,7 +391,7 @@ def _compute_is_editable(self):
         )
         is_approver = self.env.user.has_group('hr_expense.group_hr_expense_user')
         for sheet in self:
-            if sheet.state not in {'draft', 'submit', 'approve'}:
+            if sheet.state not in {'draft', 'submit'}:
```

Comment:
so an approved expense can still be modified? you sure? :suspect: 

edit: ok only for HR/admin people, so it sounds reasonable